pyiron/base/job/interactive.py

```diff
@@ -263,6 +265,66 @@ def interactive_open(self):
         """
         self.server.run_mode.interactive = True
 
+    def hd_copy(self, hdf_old, hdf_new, exclude_groups = []):
+        """
+        args:
+            hdf_old (ProjectHDFio): old hdf
+            hdf_new (ProjectHDFio): new hdf
+            exclude_groups (list): list of groups to delete
+        """
+        for p in hdf_old.list_nodes():
+            hdf_new[p] = hdf_old[p]
+        for p in hdf_old.list_groups():
+            if p in exclude_groups:
+                continue
+            h_new = hdf_new.create_group(p)
+            self.hd_copy(hdf_old[p], h_new, exclude_groups=exclude_groups)
+        return hdf_new
+
+    def file_size(self, hdf):
+        """
+        args:
+            hdf (ProjectHDFio): hdf file
+        """
+        return os.path.getsize(hdf.file_name)
+
+    def get_size(self, hdf):
+        """
+        args:
+            hdf (ProjectHDFio): hdf file
+        """
+        m = 0
+        for p in hdf.list_nodes():
+            m += sys.getsizeof(hdf[p])
+        for p in hdf.list_groups():
+            m += self.get_size(hdf[p])
+        return m
+
+    def rewrite_hdf5(self, info=False, exclude_groups=['interactive']):
```

Comment:
The interactive group might be required if we restart the execution of the job in a separate run. In addition I could see this function being implemented in pyiron.base.job.core.JobCore